core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -33,11 +33,22 @@
  */
 @ThreadSafe
 public final class PropertyKey implements Comparable<PropertyKey> {
+
+  /**
+   * The level consistency check should apply to a certain property key.
+   */
+  public enum ConsistencyCheckLevel {
+    IGNORE,
+    WARN,
+    ENFORCE,
```

Comment:
shall this be error to be consistent?

Reply:
All the enum values here are verbs indicating what action should be applied when doing consistency check. I thought about `error` but it does not sound like an action.

Reply:
oh, I see. Yeah, we can keep what you had.